nss-day-cohort-48/rare-server-orchesta-outlaws
# Vc Get All Subbed Posts

#### Changes Made

1. Added `steve-subbed-posts.sql` to the new `sql-tests` directory. Everything in there should just be SELECTs.
2. Added `steve-subs.sql` which defines three new users with a post each and makes Steve follow two of them.
3. Added `get_subbed_posts_for_user` to `posts/request.py` and exported it via `posts/__init__.py`
4. Implemented `get_subbed_posts_for_user` in `do_GET`

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout vc-get-sub-posts
   ```
2. Ensure the backend and client are both running.
3. **Run `rare-init.sql` to set up the database**
4. Test app functionality.
   > `GET /subs?follower_id=1` => returns nothing
   > Run `steve-subs.sql` to create the new data
   > `GET /subs?follower_id=1` => returns two posts

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
